sqlstore/src/main/resources/db/migration/V8501__mysql8.sql

```diff
@@ -0,0 +1,201 @@
+-- Fail fast if not running MySQL 8.0
+DELIMITER //
+CREATE PROCEDURE assertVersion()
+BEGIN
+  IF VERSION() < '8.0' OR VERSION() LIKE '%MariaDB%' THEN
+    SIGNAL SQLSTATE '45000' SET MESSAGE_TEXT = 'MySQL 8.0 required';
+  END IF;
+END//
+DELIMITER ;
+DROP PROCEDURE assertVersion;
```

Comment:
create assertVersion then immediately drop it? where does the procedure get called?